### PR TITLE
JMLL-871 Building pass single record view

### PR DESCRIPTION
**Pull request by jafacakes2011**

### CODE REVIEW

File: leavers/views/security_team.py [hunk 1]
```diff
@@ -25,84 +65,240 @@ def test_func(self):
...
+            ),
+        )
+
+        return context
+
+    def get_notifications(self) -> List[Dict[str, str]]:
+        notifications = []
+
+        # Reminder sent to Line manager
+        # TODO: Update to be the right email_id
```
Comment (jafacakes2011) [inline leavers/views/security_team.py:136]: This specific email doesn't exist yet

File: leavers/views/security_team.py [hunk 2]
```diff
@@ -25,84 +65,240 @@ def test_func(self):
...
+        notifications.append(
+            {
+                "sent": bool(line_manager_reminder_emails),
+                "name": "Automated reminder sent to Line manager",
+                "last_sent": line_manager_last_sent,
+            }
+        )
+
+        # Reminder sent to Leaver
+        # TODO: Update to be the right email_id
```
Comment (jafacakes2011) [inline leavers/views/security_team.py:158]: This specific email doesn't exist yet

**Review by rossmiller — APPROVED** [review]
LGTM!

**Comment by jafacakes2011** [comment]
This was merged in with https://github.com/uktrade/jml/pull/157

**Pull request closed without merging**